templates/cube/microcerts.html

```diff
@@ -11,83 +11,107 @@
   <div class="row u-equal-height">
     <div class="col-8">
       <h1>Your progress to CUBEdom</h1>
-      <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor.</p>
+      <p>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+        tempor.
+      </p>
     </div>
     <div class="col-4"></div>
   </div>
 </section>
 
-
 <section class="p-strip">
   <div class="u-fixed-width">
-    <h2>Microcertifications - {{user.name}}</h2>
-  </div>
-</section>
-
-<section class="p-strip--light">
-  <div class="u-fixed-width">
-    <table class="p-table--mobile-card">
+    <h2>Microcertifications</h2>
+    <table class="p-table--cube p-table--mobile-card">
       <thead>
         <tr>
-          <th>&nbsp;</th>
-          <th colspan="2" class="u-align--center">Module</th>
-          <th colspan="3" class="u-align--center">Topics</th>
-          <th colspan="2" class="u-align--center">Status</th>
-          <th colspan="2" class="u-align--center">Action</th>
+          <th>#</th>
+          <th></th>
+          <th>Module</th>
+          <th>Topics</th>
+          <th class="p-table__cell--icon-placeholder">Status</th>
+          <th class="u-align--right">Action</th>
         </tr>
       </thead>
       <tbody>
-        {% for module in modules %}
+      {% for module in modules %}
         <tr>
+          <td aria-label="Module number">{{ module.number }}.</td>
           <td aria-label="Badge">
-            <p>{{ module.badge }}</p>
+            <img
+              class="p-heading-icon__img"
```

Comment:
I'm not sure you should rely on this style to work outside of the p-heading-icon pattern. Can you use `height: 2.5rem` as an inline style for now?